chapter_appendix_math/statistics_vn.md

```diff
@@ -223,22 +225,25 @@ In this equation we do not compare to the true population value $\theta$, but in
 Thus we are not measuring how far the estimator tends to be from the true value, but instead we measuring the fluctuation of the estimator itself.
 -->
 
-*dịch đoạn phía trên*
+Việc so sánh :eqref:`eq_var_est` và :eqref:`eq_mse_est` là khá quan trọng.
+Trong công thức này thay vì so sánh tập hợp dữ liệu thực $\theta$, chúng ta sử dụng $E(\hat{\theta}_n)$, giá trị trung bình mẫu kỳ vọng.
+Do đó chúng ta không đo độ lệch của bộ ước lượng so với giá trị thực mà là độ dao động của chính nó (bộ ước lượng).
 
 
 <!--
 ### The Bias-Variance Trade-off
 -->
 
-### *dịch tiêu đề phía trên*
+### Sự đánh đổi Độ Chệch-Phương Sai
 
 <!--
 It is intuitively clear that these two components contribute to the mean squared error.  
 What is somewhat shocking is that we can show that this is actually a *decomposition* of the mean squared error into two contributions.  
 That is to say that we can write the mean squared error as the sum of the variance and the square or the bias.
 -->
 
-*dịch đoạn phía trên*
+Cả hai yếu tố trên rõ ràng đều ảnh hưởng đến trung bình bình phương sai số.
+Một điều ngạc nhiên là chúng ta có thể chứng minh trung bình bình phương sai số được phân tách từ hai thành phần đó. Điều này có nghĩa là chúng ta có thể viết trung bình bình phương sai số bằng tổng của phương sai và bình phương độ chệch.
```

Comment:
```suggestion
Một điều ngạc nhiên là chúng ta có thể chứng minh trung bình bình phương sai số được phân tách từ hai thành phần đó. 
Điều này có nghĩa là chúng ta có thể viết trung bình bình phương sai số bằng tổng của phương sai và bình phương độ chệch.
```

chỗ này "of" họ ghi sai thành "or". Thiết nghĩ anh nên chạy qua repo gốc contribute cái này cho họ luôn 😄

Reply:
Mình có contribute vài chỗ bên repo gốc, Nên chừng nào họ in sách sẽ gửi tặng mình một quyển. 😁